sensor_msgs/Image image
# if a filename is given, load the file
string filename
# if a topic name is given, wait for an image on that topic
string topic
# the person identifier
string person
---
ms_face_api/Face face